README.md

```diff
@@ -53,6 +53,19 @@ To build with clipboard support, you'll additionally need x11 bindings on Linux.
 sudo apt install xorg-dev libxcb-shape0-dev libxcb-xfixes0-dev
 ```
 
+#### nightly version
```

Comment:
Good idea, thanks for the PR! Only one nit, I wouldn't use the word "nightly" as it may suggest that it needs the rust nightly compiler - it made me think of that first. Maybe "unreleased version",  "latest unreleased" or "latest version"? Although the last is somewhat ambiguous. What do you think? Just a nit anyhow

Reply:
How about "most recent repository state"?  Although many projects call their unreleased versions "nightly", including Rust itself which is why I decided for "nightly" in the first suggestion.

Reply:
Ok, happy with nightly.